Data_Base_Homework_2/model.py

```diff
@@ -0,0 +1,74 @@
+import csv
+import os
+from logger import LOG
+
+@LOG
+def create_database():
+    header = ("id","name","surname","birthdate","grade")
```

Comment:
Скобки не нужны. Литералом кортежа, по сути, являются запятая
```suggestion
    header = "id","name","surname","birthdate","grade"
```

Reply:
А еще не забывайте, пожалуйста пробелы.